website/technischeumsetzung.md

```diff
@@ -1,17 +1,203 @@
+
+Einerseits werden beim Aufruf von *ili2db* durch den *QGIS Model Baker* Parameter automatisch im Hintergrund gesetzt und andererseits können in der Eingabemaske des *QGIS Model Baker* Parameter vom Benutzer konfiguriert werden. Zusätzlich soll nun die betreffende *Metakonfigurationsdatei* dem *ili2db* übergeben werden. Doch Parameter, die dem *ili2db* direkt übergeben werden, übersteuern die Konfigurationen des übergebenen *Metakonfigurationsfile*.
+
+*QGIS Model Baker* liest die *ili2db* Parameter aus dem *Metakonfigurationsfile*. Die Parameter, die über die Eingabemaske des *QGIS Model Baker* gesetzt werden können (wie zBs. `--strokeArcs`, `--iliMetaAttrs` (für TOML) oder auch `--models`), werden vom *Metakonfigurationsfile* in die Eingabemaske geladen. Der Benutzer kann diese nun anpassen. Der *QGIS Model Baker* übergibt dem *ili2db* Aufruf nun die *Metakonfigurationsdatei* und die Parameter aus der Eingabemaske (ob angepasst oder nicht). Falls also die Parameter in der *Metakonfigurationsdatei* aufgeführt waren, dann aber in der Eingabemaske deaktiviert wurden, werden sie dem *ili2db* als "false" übergeben.
+
+Die Parameter, die der *QGIS Model Baker* im Hintergrund setzt (wie zBs. `--createFkIdx`, `--coalesceMultiPoint`), werden weiterhin gesetzt. Doch können sie vom *Metakonfigurationsfile* übersteuert werden. Wenn aber zum Beispiel im *Metakonfigurationsfile* solche Parameter nicht erwähnt sind (weil sie möglicherweise nicht gesetzt werden sollen), dann werden sie auch nicht mit "false" übersteuert.
+
+Ausnahme bietet im *Metakonfigurationsfile* ein Setting wie `onlyUseMetaConfigParams`. Wenn dies gesetzt ist, dann sollen nur die im *Metakonfigurationsfile* konfigurierten Parameter gesetzt werden und keine anderen.
+
+### Referenzen auf andere Modelle
+
+Anhand der *ili2db* kann von einem *Metakonfigurationsfile* auf andere Modelle referenziert werden. Wenn die *Metakonfigurationsfile* der Wert `models=KbS_LV95_v1_4;KbS_Basis` enthält, dann wird dies auch in der *QGIS Model Baker* Eingabemaske so angepasst. Natürlich wird daraufhin wieder eine Suche nach möglichen *Metakonfigurationsfiles* auf UsabILIty Hub gestartet, gemäss den aktuell gesetzten Modelle. Siehe dazu auch "Import mehrerer Models und ihre Toppings"
+
+## Toppings und ihre Konfiguration
+
+*Toppingfiles* sind Files, auf die in der *Metakonfiguration* referenziert wurde und die Konfigurationsinformation des GIS Projektes enthalten. Es können also Formularkonfigurationen, Style-Attribute, Legendendarstellung sowie Datenfiles sein. Für jedes Tool können individuelle Toppingfiles verwendet werden.
+
+Vom *QGIS Model Baker* unterstützte *Toppingfiles*:
+- QML Files für Layerkonfigurationen<>
+- YAML Files für Legendendarstellung und Layerreihenfolge
+- XTF/ITF Files für Datenimport
+
+### QML
+Für Layereigenschaften wie Formularkonfigurationen, Symbologie etc. werden QML Files als *Toppingfiles* geladen. 
+
+![style](../images/style.png)
```

Comment:
Same here. This file here is in `_services/imagers/style.png` - this breaks the link.